src/app/routes/settings.index.tsx

```
@@ -0,0 +1,88 @@
+      ],
+    },
+  ];
+
+  return (
+    <Layout>
+      <SubPageHeader title='설정' back />
+      <ul className='flex flex-col list-none gap-6 pt-6 px-5'>
+        {settingGroups.map((settingGroup) => (
+          <li className='flex flex-col gap-4'>
+            <p className='text-[13px] text-[#999] font-semibold'>
+              {settingGroup.text}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)